deploy.sh

```diff
@@ -1,5 +1,6 @@
 RELEASE=`./release.sh`
 PACKAGE_NAME=application.monitoring.javascript
+ENVIRONMENT=production
```

Comment:
Noting this is fine, because the only time you set it as 'production' is when you deploy to app engine, which is what deploy.sh is for. So it matches with what is in src/index.js, should be no conflicts where sentry-cli is setting one value and src/index.js is setting a different one:

```
let ENVIRONMENT
if (window.location.hostname === "localhost") {
  ENVIRONMENT = "test"
} else { // App Engine
  ENVIRONMENT = "production"
}

...
Sentry.init({
  dsn: DSN,
  release: RELEASE,
  environment: ENVIRONMENT
...
```

Reply:
^ but interesting this is not pulling from the scripts defined in package.json, I suppose. It totally could, but `npm run build` is used for testing locally, as well as for uploading a build with the deploy script, therefore you'd sometimes want it to say 'test' (when running locally) and sometimes want it to say 'production' (when deploying), so I found it more convenient to use the window.location.hostname as the indicator to whether you're running in Test or Production.

The run.sh script is how you run locally, it uses npm run build (defined in package.json). Or run npm run start, manually.

Reply:
oh, thanks for pointing that out!

I'm finishing a new build/deploy scripts that will pull everything from a single file (either `local.env`, `staging.env`, or `production.env`) specified as a command line arg, e.g. `./deploy.sh react --env=local`  

Once it's done we can be sure all the environment variables are consistent and you don't have to keep track of local changes to effectively 4 configuration files (`.env`, `app.yaml`, `deploys.h` - stuff like this hardcoded in it, `<project>/deploys.h` that duplicates it)